modules/osd-storage-pv-aws-connect-pods.adoc

```diff
@@ -0,0 +1,37 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/
+
+[id="osd-storage-pv-aws-connect-pods_{context}"]
+= Connecting pods
+
+The PersistentVolumeClaim (PVC) that was created in your project is ready for use. You can create a sample pod to test this PVC.
```

Comment:
s/PersistentVolumeClaim/\`PersistentVolumeClaim` object

This also applies to all other uses of the term, unless "persistent volume claim" is preferred